vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailViewEvents.kt

```diff
@@ -43,6 +45,11 @@ sealed class RoomDetailViewEvents : VectorViewEvents {
     data class NavigateToEvent(val eventId: String) : RoomDetailViewEvents()
     data class JoinJitsiConference(val widget: Widget, val withVideo: Boolean) : RoomDetailViewEvents()
 
+    object OpenInvitePeople : RoomDetailViewEvents()
+    object OpenSetRoomAvatarDialog : RoomDetailViewEvents()
+    object OpenRoomSettings : RoomDetailViewEvents()
+    data class ShowRoomAvatarFullScreen(val matrixItem: MatrixItem?, val view: View?) : RoomDetailViewEvents()
```

Comment:
Same remark

Reply:
Actually I think the remark is valid for all new Events here.

Reply:
same remark